administrator/components/com_content/views/article/tmpl/edit.php

```diff
@@ -82,12 +88,20 @@
 $isModal = $input->get('layout') == 'modal' ? true : false;
 $layout = $isModal ? 'modal' : 'edit';
 $tmpl = $isModal ? '&tmpl=component' : '';
+
+JFactory::getDocument()->addScriptDeclaration('
+	var sessionToken = "' . JSession::getFormToken() . '";
+	var articleId    = "' . $this->item->id . '";
+  ');
+JHtml::script('system/share-uncompressed.js', false, true);
```

Comment:
You need to enable the Joomla debug mode and the uncompressed version will be loaded.